playground/mocks/data/idp/idx/authenticator-verification-okta-verify-reject-push.json

```diff
@@ -19,15 +23,38 @@
         "rel":[
           "create-form"
         ],
-        "name":"challenge-poll",
+        "name":"resend-push",
```

Comment:
do you only `resend-push` remediation in `poll` when user has rejected the notificaiton?

Reply:
reject or timeout

Reply:
@ManuMalhotra-okta can confirm, but this may be for any errors. Because on any error the user should have the option to resend push